indico/MaKaC/webinterface/tpls/RoomBookingSearchRoomsResults.tpl

```diff
@@ -90,3 +90,22 @@
   </tr>
 </table>
 <br />
+<script type="text/javascript">
+  $(document).ready(function() {
+      $.fn.extend({
```

Comment:
that's not a good idea, especially not in page js. why not simply call the tablesorter function directly?